ee/clickhouse/materialized_columns/test/test_columns.py

```diff
@@ -147,6 +147,18 @@ def test_backfilling_data(self, _):
             [("1", ""), ("2", "5"), ("3", ""), ("", ""), ("4", ""), ("", "6"), ("", "7")],
         )
 
+    def test_column_types(self):
+        materialize("events", "myprop")
+
+        expr = "replaceRegexpAll(JSONExtractRaw(properties, 'myprop'), concat('^[', regexpQuoteMeta('\"'), ']*|[', regexpQuoteMeta('\"'), ']*$'), '')"
```

Comment:
Where is this expression coming from? O.o

Reply:
Clickhouse expands `trim(BOTH '\"' FROM JSONExtractRaw(properties, %(key)s))` to this in the table defs.

Reply:
Worth a comment please?